content/blog/zephyr-how-to-use-psram/index.md

```diff
@@ -0,0 +1,605 @@
+---
+title: "Enabling External PSRAM for Zephyr Applications on ESP32 SoCs"
+date: 2024-12-04T00:00:00-00:00
+showAuthor: false
+featureAsset: "./psram_cover.webp"
```

Comment:
You can remove this line if you rename the `psram_cover.webp` file to `featured.webp`.